src/get-started/flutter-for/ios-devs.md

```diff
@@ -2099,159 +3271,149 @@ class _SampleAppPageState extends State<SampleAppPage> {
-            });
-          },
-          decoration: InputDecoration(
-            hintText: 'This is a hint',
-            errorText: _errorText,
-          ),
-        ),
-      ),
+      body: getBody(),
     );
   }
 }
 ```
 
-## Interacting with hardware, third party services and the platform
-
-### How do I interact with the platform, and with platform native code?
-
-Flutter doesn't run code directly on the underlying platform;
-rather, the Dart code that makes up a Flutter app is run natively
-on the device, "sidestepping" the SDK provided by the platform.
-That means, for example, when you perform a network request
-in Dart, it runs directly in the Dart context.
-You don't use the Android or iOS APIs you normally take advantage
-of when writing native apps. Your Flutter app is still hosted
-in a native app's `ViewController` as a view, but you don't
-have direct access to the `ViewController` itself,
-or the native framework.
-
-This doesn't mean Flutter apps cannot interact with those native APIs,
-or with any native code you have.
-Flutter provides the [platform channel][],
-which communicates and exchanges data with the
-`ViewController` that hosts your Flutter view.
-Platform channels are essentially an asynchronous messaging mechanism
-that bridge the Dart code with the host `ViewController`
-and the iOS framework it runs on.
-You can use platform channels to execute a method on the native side,
-or to retrieve some data from the device's sensors, for example.
-
-In addition to directly using platform channels,
-you can use a variety of pre-made [plugins][]
-that encapsulate the native and Dart code for a specific goal.
-For example, you can use a plugin to access the camera roll
-and the device camera directly from Flutter,
-without having to write your own integration.
-Plugins are found on [pub.dev],
-Dart and Flutter's open source package repository.
-Some packages might support native integrations on iOS,
-Android, web, or all of the above.
-
-If you can't find a plugin on pub.dev that fits your needs,
-you can [write your own][] and [publish it on pub.dev][].
-
-### How do I access the GPS sensor?
+### Making network requests
 
-Use the [`geolocator`][] community plugin.
-
-### How do I access the camera?
+Making a network call in Flutter is easy when you
+use the popular [`http` package][]. This abstracts
+away a lot of the networking that you might normally
+implement yourself, making it simple to make network calls.
 
-The [`camera`][] plugin is popular for accessing the camera.
+To use the `http` package, add it to your dependencies in `pubspec.yaml`:
 
-### How do I log in with Facebook?
+```yaml
+dependencies:
+  http: ^0.13.4
+```
 
-To log in with Facebook,
-use the [`flutter_facebook_login`][] community plugin.
+To make a network call,
+call `await` on the `async` function `http.get()`:
 
-### How do I use Firebase features?
+<?code-excerpt "lib/progress.dart (loadData)"?>
+```dart
+Future<void> loadData() async {
```

Comment:
Again, it seems to me that you should model best practice by including a try catch statement.